ragen/env/sudoku/env.py

```
@@ -1,540 +1,348 @@
-    
-                    # print(game_state)
-                else:
-                    game_state += "\nNo empty positions found (board might be complete)."
-            except Exception as e:
-                game_state += f"\nCould not find coordinates: {str(e)}"
-            
-            return game_state
+            return self.render(), reward, False, info
+
+        # Place the number
+        old_value = self.current_grid[row, col]
```

Ruff (0.14.7)

Local variable `old_value` is assigned to but never used

Remove assignment to unused variable `old_value`

(F841)